src/ledger/parse/transaction.ts

```diff
@@ -42,10 +42,11 @@ function parseTransactionType(type) {
   return mapping[type] || null
 }
 
-function parseTransaction(tx: any): any {
+function parseTransaction(tx: any, includeRawTransaction: boolean): any {
   const type = parseTransactionType(tx.TransactionType)
+  const _parsePayment = payment => parsePayment(payment, includeRawTransaction)
```

Comment:
It looks like `includeRawTransaction` only happens for payments.

Reply:
I think only payments have the confusing `amount` field, but it's probably not a bad idea to include the raw transaction for all transaction types.